mappings/servers/twerion/metadata.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "twerion",
+    "name": "Twerion Network",
+    "description": "A network focused on PvP",
+    "addresses": [
+        "twerion.net",
+        "twerion.de",
+        "twerion.com",
+        "twerion.eu",
+        "twerion.org"
+    ],
+    "primaryAddress": "cb.twerion.net",
```

Comment:
What’s the purpose to have a CB specific primary address? We typically accept primary addresses for the whole server, not CB specific primary addresses.

Reply:
Join tracking, so we know exactly how many people are joining from the Cheatbreaker launcher.

Reply:
We can also change it if desired.

Reply:
The proper way to track CheatBreaker users is through the https://github.com/CheatBreakerNet/Bukkit-API. We have a dedicated plugin message you can use to check.

Besides, end users on other clients can just use this address to skew your intended tracking system

I'd advise just using the normal address instead if the justification is just this improper tracking system

Reply:
Thanks for the quick reply, we will do it via CheatBreaker API.

Reply:
We have changed it back to the normal domain.